apps/pie-docs/src/_includes/home-page.njk

```diff
@@ -0,0 +1,30 @@
+{% set heroSrc = ['/assets/img/heroes/', heroImageSrc] | join | url %}
+
+<div class="c-homePage" data-test-id="site_content">
+    <div class="c-homePage-header">
+        <div class="c-content-container">
```

Comment:
Same here with the classnames – I think these need to be tidied up a little to be a bit clearer.